services/gateway/apis/http-version3/swagger/definitions/networkStatus.json

```diff
@@ -156,16 +152,7 @@
                                         "port": 7667
                                     }
                                 }
-                            },
-                            "blacklistedIPs": {
-                                "type": "integer",
-                                "example": 15360
-                            },
-                            "fixedPeers": {
-                                "type": "integer",
-                                "example": 10
-                            },
-                            "whitelistedPeers": {}
```

Comment:
Need to double check if it was correct before.

Reply:
Its not a part of API response and even not included in test cases:
https://github.com/LiskHQ/lisk-service/blob/1544-fix-swagger-specs/tests/schemas/api_v3/networkStatus.schema.js